lxd/instance/drivers/driver_lxc.go

```diff
@@ -2657,6 +2657,11 @@ func (d *lxc) Stop(stateful bool) error {
 		return err
 	}
 
+	err = d.VolatileSet(map[string]string{"volatile.last_state.ready": "false"})
```

Comment:
This call and the one in `Shutdown` are problematic because they won't catch instances that self-shutdown.
Additionally there is already a call to `VolatileSet` in `onStop()` (where this should also be), so we will be able to combine them into a single query to avoid additional DB queries.

Please remove from here and `Shutdown` and add to where `"volatile.last_state.power": "STOPPED"` is set in `onStop()`.